commit 4df7f804130cfb8dda2648e8f22e9c97ff595cc7
Author: Michael Ellerman <mpe@ellerman.id.au>
Date:   Fri Mar 17 00:48:33 2017 +0000

    powerpc/mm: Fix build break when CMA=n && SPAPR_TCE_IOMMU=y
    
    [ Upstream commit a05ef161cdd22faccffe06f21fc8f1e249565385 ]
    
    Currently the build breaks if CMA=n and SPAPR_TCE_IOMMU=y:
    
      arch/powerpc/mm/mmu_context_iommu.c: In function ‘mm_iommu_get’:
      arch/powerpc/mm/mmu_context_iommu.c:193:42: error: ‘MIGRATE_CMA’ undeclared (first use in this function)
      if (get_pageblock_migratetype(page) == MIGRATE_CMA) {
      ^~~~~~~~~~~
    
    Fix it by using the existing is_migrate_cma_page(), which evaulates to
    false when CMA=n.
    
    Fixes: 2e5bbb5461f1 ("KVM: PPC: Book3S HV: Migrate pinned pages out of CMA")
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1451_4df7f804130cfb8dda2648e8f22e9c97ff595cc7.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1451_4df7f804130cfb8dda2648e8f22e9c97ff595cc7.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
